### ISSUE TITLE
More Sounds

### ISSUE THREAD

**Issue by Gegy** [FiskFille/TransformersMod#80]
Should we add more sounds like drifting sounds?

**Comment by FiskFille** [comment]
Yes. I have recently figured out how to cancel, pause and stop sounds that are already in progress, so this will indeed be something interesting to work on! :D
